SpeedyContactAdd WatchKit Extension/Controller/InterfaceController.swift

```diff
@@ -0,0 +1,202 @@
+    }
+    
+    @IBOutlet var lblPhoneNumber: WKInterfaceLabel!
+    @IBOutlet weak var lblReName: WKInterfaceLabel!
+    
+    var session:WCSession!
+    
+    var contacts: [Contacts] = []
+    var numberStore = String()
+    var recipNameString = String()
+    var fullNameString = "fullNameString"
+    var numberString = "numberString"
+    var timer = Timer()
+    let contactsDefault = UserDefaults.standard
+    let checkIfUserDefaultExist:((String) -> Bool) = { key in
+        return UserDefaults.standard.object(forKey: key) != nil
+    }
+    
+    func numberAdd(myCharacter:String) {
+        numberStore += myCharacter
+        lblPhoneNumber.setText(numberStore)
+    }
+    func clearItems() {
+        numberStore = ""
+        recipNameString = ""
+        lblReName.setText("Contact Name")
+        lblPhoneNumber.setText("")
+    }
+    
+    //Mark: Keypad that addes numbers to phone number
+    @IBAction func btnNumber1Action(){
+        numberAdd(myCharacter: "1")
+    }
+    @IBAction func btnNumber2Action() {
+        numberAdd(myCharacter: "2")
+    }
+    @IBAction func btnNumber3Action() {
+        numberAdd(myCharacter: "3")
+    }
+    @IBAction func btnNumber4Action() {
+        numberAdd(myCharacter: "4")
+    }
+    @IBAction func btnNumber5Action() {
+        numberAdd(myCharacter: "5")
+    }
+    @IBAction func btnNumber6Action() {
+        numberAdd(myCharacter: "6")
+    }
+    @IBAction func btnNumber7Action() {
+        numberAdd(myCharacter: "7")
+    }
+    @IBAction func btnNumber8Action() {
+        numberAdd(myCharacter: "8")
+    }
+    @IBAction func btnNumber9Action() {
+        numberAdd(myCharacter: "9")
+    }
+    @IBAction func btnNumber0Action() {
+        numberAdd(myCharacter: "0")
+    }
+    @IBAction func btnBackspaceAction() {
+        if numberStore != "" {
+            numberStore.remove(at: numberStore.index(before: numberStore.endIndex))
+            lblPhoneNumber.setText("\(numberStore)")
+        } else {return}
+    }
+    // MARK: - Force Push button resets all
+    @IBAction func btnMenuClear() {
+        clearItems()
+    }
+    
+    // MARK: - Add name to RecipNameStrings
+    @IBAction func btnAddNameAction() {
+        presentTextInputController(withSuggestions: ["New Contact"], allowedInputMode: WKTextInputMode.plain) { (result) in
+            guard let result = result else {return}
+            let resultString = result[0] as! String
+            self.recipNameString = resultString
+            self.lblReName.setText("\(resultString)")
+        }
+    }
+    
+    
+    
+    //  MARK: Send user info to iPhone. If either name or number are blank, do nothing.
+    @IBAction func btnSendtoPhone() {
+        
+        if recipNameString == "" || numberStore == "" {return}
+        
+        // TODO: - Add in a notification that states one of these is blank.
+        
+        if (WCSession.default.isReachable) {
+             if checkIfUserDefaultExist("SavedContacts") == true {
```

Comment:
```suggestion
             if checkIfUserDefaultExist("SavedContacts") {
```

No need to check if something is `== true`